apis/beacon/blobs/post_blobs.yaml

```diff
@@ -0,0 +1,54 @@
+post:
+  tags:
+    - Beacon
+    - ValidatorRequiredApi
+  summary: "Publish a signed blob."
+  operationId: "publishBlob"
+  description: |
+    Instructs the beacon node to broadcast a newly signed blob to the beacon network,
+    to be included in the beacon chain. A success response (20x) indicates that the blob
+    passed gossip validation and was successfully broadcast onto the network.
+    The beacon node is also expected to integrate the blob into state once it receives the 
+    corresponding block and any other corresponding blobs, but may broadcast it
+    before doing so, so as to aid timely delivery of the block and blobs. Should the blob fail full
+    validation, a separate success response code (202) is used to indicate that the blob was
+    successfully broadcast but failed integration.
+  parameters:
+    - in: header
+      schema:
+        $ref: '../../../beacon-node-oapi.yaml#/components/schemas/ConsensusVersion'
+      required: false
+      name: Eth-Consensus-Version
+      description: "Version of the blob being submitted, if using SSZ encoding."
+  requestBody:
+    description: "A variant of `SignedBlobSidecar` object composed of a `BlindedBlobSidecar` object (produced by beacon node) and validator signature."
+    required: true
+    content:
+      application/json:
+        schema:
+          oneOf:
+            - $ref: "../../../beacon-node-oapi.yaml#/components/schemas/Deneb.SignedBlindedBlobSidecar"
+      application/octet-stream:
+        schema:
+          description: "SSZ serialized blob sidecar bytes. Use content type header to indicate that SSZ data is contained in the request body."
+  responses:
+    "200":
+      description: "The blob was validated successfully and has been broadcast. It has also been integrated into the beacon node's database."
+    "202":
+      description: "The blob could not be integrated into the beacon node's database as it failed validation, but was successfully broadcast."
+    "400":
+      description: "The `SignedBlobSidecar` object is invalid and could not be broadcast"
```

Comment:
Shouldn't be `SignedBlindedBlobSidecar` instead of `SignedBlobSidecar`?

Reply:
i guess description is talking about constructed object